docs/installation-and-operation/serialization.md

```diff
@@ -4,53 +4,229 @@ redirect_from:
+```
+
+## Before exporting or importing
+
+If your Metabase is:
+
+- currently running, and
+- you're using the default H2 database
+
+You'll need to stop Metabase before exporting or importing.
+
+Otherwise, you're good.
+
+## Exporting a Metabase
+
+To export the contents of a Metabase instance, use the following command in your terminal:
+
+```
+java -jar metabase.jar export [export_name]
+```
+
+## Export options
+
+To view a list of `export` options, use the `help` command:
+
+```
+java -jar metabase.jar help export
+```
+
+Which will print something like:
+
+```
+export path & options
+	 Serialize Metabase instance into directory at `path`.
+	 Options:
+	   -u, --user EMAIL                Include collections owned by the specified user
+	   -c, --collection ID             Export only specified ID; may occur multiple times.
+	       --collections ID_LIST       (Legacy-style) Export collections in comma-separated list of IDs, e.g. '123,456'.
+	   -C, --no-collections            Do not export any content in collections.
+	   -S, --no-settings               Do not export settings.yaml
+	   -D, --no-data-model             Do not export any data model entities; useful for subsequent exports.
+	   -f, --include-field-values      Include field values along with field metadata.
+	   -s, --include-database-secrets  Include database connection details (in plain text; use caution).
+```
+
+### `--user`
+
+The `user` flag tells Metabase to include collections owned by the specified user, identified by email. QUESTION: What about accounts created with SSO?
+
+### `--collection`
+
+The `--collection` flag (alias `-c`) lets you specify by ID one or more collections to include in the export. You can find the collection ID in the collection's URL, e.g., for a collection at: `your-metabase.com/collection/42-terraforming-progress`, the ID would be `42`.
+
+If you want to specify multiple collections, separate the IDs with commas. E.g.,
+
+```
+java -jar metabase.jar export --collection 1,3,5,9
+```
+
+### `--no-collection`
+
+The `--no-collection` flag (alias `-C`) allows you to exclude one or more collections. E.g.:
+
+```
+java -jar metabase.jar export --no-collection 1,3,5,9
+```
+
+The above command would export every collection except collections with IDs `1`, `3`, `5`, and `9`.
+
+### `--no-settings`
+
+The `--no-settings` flag (alias `-S`) tells Metabase to exclude the `settings.yaml` file that includes site-wide settings, which is exported by default.
+
+### `--no-data-model`
+
+The `no-data-model` flag tells Metabase to exclude the data model settings from the export. Admins define the data model settings in the [Data model](../data-modeling/metadata-editing) tab of the Admin settings.
+
+### `--include-field-values`
+
+The `include-field-values` tells Metabase to include the sample values for field values, which Metabase uses to present dropdown menus. By default, Metabase excludes these sample field values.
+
+### `--include-database-secrets`
```

Comment:
I think it is interesting to add that people can manually add these in the target instance (in the UI)